packages/dev/docs/src/syntax-highlight.css

```diff
@@ -22,7 +21,21 @@
   --hljs-class-color: var(--spectrum-global-color-yellow-600);
   --hljs-function-color: var(--spectrum-global-color-blue-600);
   --hljs-variable-color: var(--spectrum-global-color-yellow-600);
-  --hljs-title-color: var(--spectrum-global-color-indigo-600);
+  --hljs-title-color: var(--spectrum-global-color-indigo-500);
+  --hljs-comment-color: var(--spectrum-global-color-gray-700)
+}
+
+[style*="color-scheme: light"] {
+  --hljs-string-color: #398241;
+  --hljs-literal-color:  #a9621b;
+  --hljs-attribute-color: #a9621b;
+  --hljs-class-color: #847323;
+  --hljs-variable-color: #847323;
```

Comment:
hmm, should we bring these up with spectrum? the colors are what they suggested.

Reply:
I used the leonardo tool to determine a compliant value of the same hue. We don’t have compliant values for orange, yellow, etc in Spectrum, because they’re not really meant for text. highlight.js does have a couple accessible themes.

Reply:
ok, we're going to check with spectrum just to make sure we also match spectrum-css.